pkg/apis/serving/k8s_validation_test.go

```diff
@@ -34,16 +35,9 @@ import (
 
 type configOption func(*config.Config) *config.Config
 
-func withMultiContainer() configOption {
+func withFeature(feature string) configOption {
 	return func(cfg *config.Config) *config.Config {
-		cfg.Features.MultiContainer = config.Enabled
-		return cfg
-	}
-}
-
-func withFieldRef() configOption {
-	return func(cfg *config.Config) *config.Config {
-		cfg.Features.PodSpecFieldRef = config.Enabled
+		reflect.Indirect(reflect.ValueOf(&cfg.Features)).Elem().FieldByName(feature).Set(reflect.ValueOf(config.Enabled))
```

Comment:
I feel like if folks hold this wrong (even a typo) they're gonna have a really bad time.

Reply:
Agreed but it's test code only and reduces boiler plate. If you'd still rather I spell each function out then I don't mind.

Reply:
let's spell it out for now.

I can imagine a simpler solution where each feature is a package constant with some helper methods to apply states on configs.

Reply:
ok, will do.